data-prepper-plugins/aws-plugin/src/main/java/org/opensearch/dataprepper/plugins/aws/AwsSecretsPluginConfigValueTranslator.java

```diff
@@ -11,7 +11,8 @@
 import java.util.regex.Pattern;
 
 public class AwsSecretsPluginConfigValueTranslator implements PluginConfigValueTranslator {
-    static final String AWS_SECRETS_PREFIX = "aws_secrets";
+    static final String DEPRECATED_AWS_SECRETS_PREFIX = "aws_secrets";
+    static final String AWS_SECRETS_PREFIX = "aws:secrets";
```

Comment:
The `:` is meant to differentiate between the type of variable and the variable itself. I'm not sure if this change is adding value here. It becomes hard to differentiate between the keyword and the components because we are not using a `:` as part of a keyword.

For example, with environment variables we are looking for: `env:MY_ENVIRONMENT` and for Docker `docker:some_secret_value`.

Reply:
Discussed offline. I used a slightly different approach in prefix matching but it still injects log message to indicate whether there is match from translator on the prefix.